skimage/external/tifffile/tifffile.c

```diff
@@ -10,13 +10,14 @@ Refer to the tifffile.py module for documentation and tests.
 :Organization:
   Laboratory for Fluorescence Dynamics, University of California, Irvine
 
-:Version: 2016.04.13
+:Version: 2017.01.10
 
 Requirements
 ------------
 * `CPython 2.7 or 3.5 <http://www.python.org>`_
-* `Numpy 1.10 <http://www.numpy.org>`_
+* `Numpy 1.11 <http://www.numpy.org>`_
 * A Python distutils compatible C compiler  (build)
+* `stdint.h <https://github.com/chemeris/msinttypes/>`_ for msvc9 compiler
```

Comment:
It looks like we need to include these headers on Appveyor, but I have no idea how to do that.